content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -504,6 +504,22 @@ stops adjusting the target (and sets the `ScalingActive` Condition on itself
 to `false`) until you reactivate it by manually adjusting the target's desired
 replica count or HPA's minimum replica count.
 
+## Recommended Deployment Configuration
+
+When an HPA is enabled, it is recommended that the value of `spec.replcas` of
```

Comment:
When an HPA is enabled, it is recommended that the value of `spec.replicas` of

Reply:
```suggestion
When an HPA is enabled, it is recommended that the value of `spec.replicas` of
```